src/Redux/Reducers/reservationsSlice.js

```diff
@@ -0,0 +1,35 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import axios from 'axios';
+
+export const fetchReservations = createAsyncThunk('reservations/fetchReservations', async () => {
+  const response = await axios.get('http://localhost:4000/api/v1/reservations');
+  return response.data;
+});
+
```

Comment:
This endpoint is not ready yet, I think you cannot use it unless it is ready. Morover, when are there 2 reservation slices?

Reply:
I'm working on those now. Thanks